### DISCUSSION TITLE
Can't send application because name differences

### DISCUSSION THREAD

**Discussion by Momen-MKadry** [community/community#59824] [GitHub Education]
### Hi everyone,

Ok so I was applying for the student pack now and I uploaded a screenshot of my profile on the university portal and it has my full name in my native language (Arabic) after clicking on Process my application it gave me an error that says this

> Your application cannot be reviewed until you fix the following:
You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio. The verification process looks at your GitHub profile to cross reference the evidence submitted. Do you use a variation of your name or a nickname? Is your name in non-english? When you update your profile information please log out and log back into GitHub before re-applying.

I tried to change my name in my profile as it suggested to my fill name a it is in the portal but still the same error.

**Comment by samus-aran** [comment]
Hi @Momen-MKadry 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍
